src/main/kotlin/org/hyacinthbots/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -580,7 +580,7 @@ class TemporaryModeration : Extension() {
 						return@action
 					}
 
-					everyoneRole.edit {
+					everyofneRole.edit {
```

Comment:
Build fails because of this, just fix that and then perfect

Reply:
Fixed. As it turns out if you press F without CRTL it actually writes the character somewhere in the file. Who would have thought.